src/masternodes/mn_rpc.cpp

```diff
@@ -918,12 +918,18 @@ UniValue tokenToJSON(DCT_ID const& id, CTokenImplementation const& token, bool v
         tokenObj.pushKV("isLPS", token.IsPoolShare());
         tokenObj.pushKV("finalized", token.IsFinalized());
 
-        tokenObj.pushKV("creationTx", token.creationTx.ToString());
-        tokenObj.pushKV("creationHeight", token.creationHeight);
-        tokenObj.pushKV("destructionTx", token.destructionTx.ToString());
-        tokenObj.pushKV("destructionHeight", token.destructionHeight);
         /// @todo tokens: collateral address/script
 //      tokenObj.pushKV("collateralAddress", token.destructionHeight);
+        if (id >= CTokensView::DCT_ID_START) {
```

Comment:
mismerged here

Reply:
don't need that "if" and casting to CTokenImplementation
all the difference (with applyed "minted" PR) - `tokenObj.pushKV("minted",` should appear

Reply:
@DeFiDev thanks for spotting that, can you fix this?

thank you

Reply:
ok